orca/service_test.go

```diff
@@ -151,11 +147,15 @@ func (s) TestE2E_CustomBackendMetrics_OutOfBand(t *testing.T) {
 		default:
 		}
 
+		mu.Lock()
+		expectedUtilization := float64(requests) * 0.01
```

Comment:
why did you change this value to float64(requests) * 0.01? it was `numRequests * 0.01`. The numRequests is a constant so that's we should have in want. requests are being calculated within stub server call and may not be synchronized. You shouldn't need mutex to assign a constant value